_layouts/post.html

```diff
@@ -37,8 +49,20 @@ <h1 class="title page-title">{{page.title}}</h1>
     <div class="generated-content">
         {{content}}
     </div>
+
+<hr class="divider" />
 </article>
 
+
+<div class = "social-buttons" style="text-align: center">
+Enjoyed the read? Share on social media! <br>
+<a class="fb-share-button"  data-layout="button" data-size="large" target="_blank" href="" class="fb-xfbml-parse-ignore">Share</a>
```

Comment:
Is there supposed to be a blank `href` here?

Reply:
Yep! The href is appended to the site's url which is odd but that's how the script functions. When I put anything here, it would append it to the post's url which led to a 404 error. By keeping it empty, the link redirects to the blog post.